modules/ROOT/pages/clustering/setup/routing.adoc

```diff
@@ -34,6 +34,32 @@ As a result, unevenly distributed leaderships may cause write queries to be disp
 By default, Neo4j avoids this by automatically transferring database leaderships so that they are evenly distributed throughout the cluster.
 Additionally, Neo4j automatically transfers database leaderships away from instances where those databases are configured to be read-only using xref:configuration/configuration-settings.adoc#config_server.databases.read_only[server.databases.read_only] or similar.
 
+[[clustering-client-side-routing]]
+== Client-side routing
+
+Client-side routing is when the database client takes control over which cluster member to send specific requests to.
+Typically this would be to make sure that write operations are sent to the server that can write for your specific database, and that read operations are sent to other servers.
+
+Client-side routing is based on getting a routing table from a cluster member, and then using that information to make the routing decisions.
+A routing table provides the writers, readers and routers for a specific database.
+There will usually be one writer, though there may be none if the database is read only or unhealthy.
+With the default configuration all other servers that host the database will be considered readers, i.e. the writer will not be in the list of readers.
+This is to let it focus on the write load and not have to manage two kinds of interaction.
+Typically all members of the cluster will be given as 'routers', which are servers that can be contacted to get a new routing table.
```

Comment:
Have you checked that? Are routers not only members whihc actually host the db?
I agree it could be everyone, but I don't we chnaged this.

Reply:
You are quite right, I thought we had changed that, but it's just the hosts. Will update, thanks.